@xen-orchestra/web/src/components/vm/dashboard/VmDashboardVdiUsageChart.vue

```diff
@@ -69,19 +69,27 @@ const vdiUsage = computed<LinearChartData>(() => {
 })
 
 const maxValue = computed(() => {
-  const values = vdiUsage.value.flatMap(series => series.data.map(item => item.value))
+  const values = vdiUsage.value.reduce(
+    (acc, series) => [...acc, ...series.data.map(item => item.value ?? 0)],
+    [] as number[]
+  )
 
   if (values.length === 0) {
     return 100
   }
 
-  const maxUsage = Math.max(...values)
+  const maxUsage = Math.max(...values) * 1.2
 
-  return Math.ceil(maxUsage / 50) * 50
+  return Math.ceil(maxUsage / 100) * 100
 })
 
-const byteFormatter = (value: number) => {
+const byteFormatter = (value: number | null) => {
+  if (value === null) {
+    return ''
+  }
+
   const result = formatSizeRaw(value, 1)
+
   return `${result?.value}${result?.prefix}`
```

Comment:
add space to match with VmDashboardQuickInfo.vue:36
```suggestion
  return `${result?.value} ${result?.prefix}`
```